.eslintignore

```diff
@@ -28,6 +28,8 @@ packages/server/lib/scaffold/support/commands.js
 packages/server/test/support/fixtures/projects/e2e/cypress/integration/stdout_exit_early_failing_spec.js
 packages/server/test/support/fixtures/projects/e2e/cypress/integration/browserify_typescript_failing_spec.ts
 
+**/.projects
+**/*.d.ts
```

Comment:
weird why

Reply:
in case you don't clear out the temporary .projects directory, eslint will try to get it's hands on it

and eslint cannot format .d.ts files, just throws errors